kit/ChildSession.cpp

```diff
@@ -822,6 +830,38 @@ bool ChildSession::loadDocument(const StringVector& tokens)
     return true;
 }
 
+// attempt to shutdown threads, fork and execute in the background
+bool ChildSession::saveDocumentAsync(const StringVector &tokens)
+{
+#if MOBILEAPP
```

Comment:
Perhaps we can do this above, at `static bool doBgSave = !!getenv("COOL_BGSAVE");`. It should make this already-complex function simpler.